### PR TITLE
C/cross join docs

### PR DESCRIPTION
**Pull request by celaus**
Documentation fixes for CROSS JOIN.

### CODE REVIEW

File: docs/sql/queries.txt [hunk 1]
```diff
@@ -48,9 +48,10 @@ Aliases can be used to change the output name of the columns::
 FROM Clause
 -----------
 
-The ``FROM`` clause is used to reference the table to query. A table is referenced
-by schema and table name and can optionally be aliased. If the table ``t`` is
-only referenced by table name, crate assumes the table ``doc.t`` was meant.
+The ``FROM`` clause is used to reference one or two tables.
```
Comment (chaudum) [inline docs/sql/queries.txt]: is it really limited to max 2 tables ATM?

File: docs/sql/queries.txt [hunk 2]
```diff
@@ -48,9 +48,10 @@ Aliases can be used to change the output name of the columns::
...
+A table is referenced by schema and table name and can optionally be aliased.
+If the table ``t`` is only referenced by table name, crate assumes the table
```
Comment (chaudum) [inline docs/sql/queries.txt]: typo: Crate

File: docs/sql/reference/select.txt [hunk 3]
```diff
@@ -91,15 +91,34 @@ FROM Clause
...
 The FROM clause specifies the source table for the SELECT::
 
-    FROM table_ident [ [AS] table_alias ]
+    FROM table_reference [{CROSS JOIN | , } table_reference [{CROSS JOIN | ,} ...]]
 
-:table_ident: The name (optionally schema-qualified) of an existing table.
+Table Reference
+...............
+
+A table Reference is a table ident with an optional table alias::
```
Comment (mikethebeer) [inline docs/sql/reference/select.txt]: table reference

File: docs/sql/queries.txt [hunk 4]
```diff
@@ -89,6 +90,60 @@ A table can be aliased for the sake of brevity too::
...
     SELECT 1 row in set (... sec)
 
+
+.. _sql_dql_cross_join:
+
+Cross Joins
+...........
+
+Referencing two tables results in a ``Cross Join``. The result contains
+every possible combination of rows from the given tables ( T1 * T2 * ...)::
```
Comment (chaudum) [inline docs/sql/queries.txt]: every possible combination (**Cartesian product**) of ...

File: docs/sql/queries.txt [hunk 5]
```diff
@@ -397,7 +452,7 @@ NOT
 
     [ NOT ] boolean_expression
 
-The result type is boolean 
+The result type is boolean
```
Comment (chaudum) [inline docs/sql/queries.txt]: `.`

File: docs/sql/reference/select.txt (hunk 3 above)
Comment (chaudum) [inline docs/sql/reference/select.txt]: typo: **r**eference

File: docs/sql/reference/select.txt [hunk 6]
```diff
@@ -91,15 +91,34 @@ FROM Clause
...
+
+    table_ident [ [AS] table_alias ]
+
+:table_ident: The name (optionally schema-qualified) of an existing table
```
Comment (chaudum) [inline docs/sql/reference/select.txt]: I prefer `.` at the end of descriptions, even if they are not full sentences. geschmacksache ...

File: docs/sql/queries.txt (hunk 1 above)
Comment (celaus) [inline docs/sql/queries.txt]: unfortunately yes :no_good: .

**Comment by chaudum** [comment]
Wait for green travis :)

**Pull request closed without merging**